frontend/apps/web/src/components/system/ProgressBarWrapper.tsx

```diff
@@ -2,14 +2,17 @@ import { CardLayout } from '@filecoin/ui'
 import React from 'react'
 import styled from 'styled-components'
 
+export interface ProgressBarWrapperProps {
+  shadow: boolean
+  children: React.ReactNode
+}
 
-const ProgressBarWrapper = ({shadow=false,children,...props}) => {
-    return (
-        <Wrapper shadow={shadow}>
-          {children}
-        </Wrapper>
-    )
-
+const ProgressBarWrapper: React.FC<ProgressBarWrapperProps> = ({
+  shadow = false,
+  children,
+  ...props
```

Comment:
If we don't use ...props, we should remove it

Reply:
Removed, thanks.